.github/workflows/ci-build.yaml

```diff
@@ -23,9 +23,20 @@ jobs:
       uses: actions/setup-python@v2
       with:
         python-version: ${{ matrix.python-version }}
-    - name: Install dependencies
+    - name: Get pip cache dir
+      id: pip-cache
       run: |
         python -m pip install --upgrade pip
+        echo "::set-output name=dir::$(pip cache dir)"
+    - name: pip cache
+      uses: actions/cache@v2
+      with:
+        path: ${{ steps.pip-cache.outputs.dir }}
+        key: ${{ runner.os }}-pip-${{ hashFiles('**/test-requirements.txt') }}
```

Comment:
Should we hash the doc requirements file somewhere as well?

Reply:
Done. I decided to use the same cache key for all, despite the requirements not 100% overlapping. I think it will make things more efficient.